cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/packages.proto

```diff
@@ -70,12 +70,21 @@ message GetAvailablePackageSummariesRequest {
   // The context (cluster/namespace) for the request
   Context context = 1;
 
+  // FilterOptions
+  //
   // FilterOptions available when requesting summaries
   message FilterOptions {
```

Comment:
Adding these comments for a better swagger document. 
When embedding these messages inside other msgs we often forgot to add the proper docs, that's why I tend not to embed them. But, on the other side, it's clearer to know which mgs they belong to, so... no strong opinion though.

Reply:
Yep, and better if we then move the embedded message out. But again, if it's OK with you, I'll add those in the separate PR which hasn't landed yet which adds the filter options. It'll be easier if you don't merge that here nor add functionality depending on them in this PR, but rather keep this one focused on getting the available package summaries (securely, checking the namespace) without filter options. Just so we can work together without depending on unlanded branches.

Reply:
Or if you want to update once it's landed and still include the filters, that's fine too, but just keen to avoid depending on other people's unlanded code unless it's necessary (ie. if this PR was dependent on filtering options, but I don't think it is).

Reply:
Hah - so I just found the source of confusion here. Sorry, it was my fault. I'd thought I'd created a PR with the filter options yesterday... but it turns out I'd created the branch before our standup but didn't get back to push it and create the PR. I only found out because I just went to update my branch and pushed the changes to find that it created a new branch on GH. So from your POV you obviously weren't working on something that I'd done at all. Sorry for the confusion :)

Reply:
Hahah, I wasn't getting the point here; I added the FilterOptions in my branch so, as you said, I didn't see you were also working on that. It totally makes sense now.
I'll add the remaining changes to the filter (version and app version) here so that we have the whole filtering functionality implemented.